config/cmake/mccacheinit.cmake

```diff
@@ -16,11 +16,9 @@ set (HDF4_BUILD_UTILS ON CACHE BOOL "Build HDF4 Library Utilities" FORCE)
 
 set (HDF4_BUILD_GENERATORS OFF CACHE BOOL "Build Test Generators" FORCE)
 
-set (HDF4_ENABLE_JPEG_LIB_SUPPORT ON CACHE BOOL "Use JPEG Filter" FORCE)
```

Comment:
Is this JPEG line removed intentionally?

Reply:
yes, it is on by default